curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -91,6 +91,9 @@ void UnInitFuseClient() {
 
 void FuseOpInit(void *userdata, struct fuse_conn_info *conn) {
     g_ClientInstance->FuseOpInit(userdata, conn);
+    if (conn->capable & FUSE_CAP_WRITEBACK_CACHE) {
```

Comment:
we decide use FUSE_CAP_WRITEBACK_CACHE？

Reply:
> we decide use FUSE_CAP_WRITEBACK_CACHE？

Not yet, this is used to test before. I will delete this.